sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -734,6 +1002,9 @@ export class TextAnalysisClient {
     analyze<ActionName extends "LanguageDetection">(actionName: ActionName, documents: string[], countryHint?: string, options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
     analyze<ActionName extends AnalyzeActionName = AnalyzeActionName>(actionName: ActionName, documents: TextDocumentInput[], options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
     analyze<ActionName extends AnalyzeActionName = AnalyzeActionName>(actionName: ActionName, documents: string[], language?: string, options?: AnalyzeActionParameters<ActionName> & TextAnalysisOperationOptions): Promise<AnalyzeResult<ActionName>>;
+    beginAnalyzeBatch(actions: AnalyzeBatchAction[], documents: string[], language?: string, options?: BeginAnalyzeBatchOptions): Promise<AnalyzeBatchPoller>;
```

Comment:
is it worth having an overload where you can pass a single `AnalyzeBatchAction` and avoid the surrounding `[]`? Maybe that causes more effort (and overhead) than is necessary. Certainly we could always add it later if desired.

Reply:
Yeah, I can see that as a plausible scenario since some actions here are unique (not supported by the faster sibling `analyze`). I agree that we could wait for customer feedback before we add it.

Reply:
Thanks for the context! I agree having four overloads per operation could be a lot especially in the older client version where there are many operations already.